### ISSUE TITLE
Trilinos requires CMake 3.10.0

### ISSUE THREAD

**Issue by ibaned** [gahansen/Albany#371]
Most of our dashboard is red this morning because Trilinos switched its minimum requirement to 3.10. To be fair, I did mention this is a couple of Albany telecons already.

@gahansen 
@ikalash 
@lxmota

**Comment by ikalash** [comment]
I fixed all the ones I own except Edison.  Edison does not have a high enough cmake module.  I've asked them to install it.  I can just build it locally but I think enough people use Trilinos on NERSC to warrant creating a module.

**Comment by ibaned** [comment]
I fixed the `lcm-cee` module, so now most builds are green again. the last few will probably get done next week

**Comment by ikalash** [comment]
There are still a number of failing builds on the CDash site due to not having a high enough cmake, namely the ones names "Trilinos", "TrilinosClang", "TrilinosIntel" and "TrilinosDbg".  @gahansen I believe you are the owner of these builds - can you please switch to a newer cmake so that they run again?

**Comment by gahansen** [comment]
Yes, I own these builds - thanks for bringing this to my attention. I'll work on these and get them running again.

**Comment by ikalash** [comment]
It looks like all the nightly builds have switched to the new cmake, therefore closing the issue.
